### PR TITLE
genesis validators

### PR DESCRIPTION
**Pull request by rob-solana**
#### Problem
 genesis validators aren't yet in the code for genesis

 #### Summary of Changes
 add support for provisioning genesis validators

Fixes #

### CODE REVIEW

File: genesis/src/genesis_accounts.rs [hunk 1]
```diff
@@ -614,6 +615,21 @@ fn add_stakes(
         .sum::<u64>()
 }
 
+pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[ValidatorInfo {
+    name: "aurel@ethereum.ro",
+    node: "GeZ5PrJi9muVCJiJAaFBNGoCEdxGEqTp7L2BmT2WTTy1",
+    vote: "7ZdRx2EBYoRuPfyeoNbuHodMUXcAQRcC37MUw3kP6akn",
+    node_sol: 500.0,
+    commission: 50,
```
Comment (mvines) [inline genesis/src/genesis_accounts.rs]: 0% commission for SLP1 no?

Comment (rob-solana) [reply genesis/src/genesis_accounts.rs]: open question, I think @ericlwilliams 

since no rewards for SLP1, commission is somewhat irrelevant

Comment (mvines) [reply genesis/src/genesis_accounts.rs]: Can we just set 0 so we can keep moving and revisit later if need.

Comment (rob-solana) [reply genesis/src/genesis_accounts.rs]: done

**Pull request merged**